documentation/documentation_publisher.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-console */
+import AWS from 'aws-sdk';
+import {Command} from 'commander';
+import S3 from 'aws-sdk/clients/s3';
+import * as fs from 'fs';
+import * as mime from 'mime';
+import * as path from 'path';
+
+const AwsRegion = 'us-west-2';
+const BaseGeneratedDocsPath = 'site';
+const DocumentationBucket = 'developer-documentation';
+const PacksSdkBucketRootPath = 'packs';
+const EnvironmentKeyRoot = `${PacksSdkBucketRootPath}/current`;
+const Environments = ['adhoc', 'head', 'staging', 'prod'];
+const program = new Command();
+
+// only works after `program` has been parsed
+function isVerbose() {
+  return program.opts().verbose;
+}
+
+function handleError(e: Error) {
+  console.error(e);
+  process.exit(1);
```

Comment:
We have a printAndExit utility that essentially does this, if you want to reuse.